src/Compilers/CSharp/Test/Semantic/Semantics/ConstantTests.cs

```diff
@@ -3462,6 +3462,33 @@ static void Main()
             source.Append(source1);
             CompileAndVerify(source.ToString(), expectedOutput: "58430604");
         }
+
+        [Fact]
+        public void ConstantInterpolatedStringsSimple()
```

Comment:
Please add a prototype comment for testing invalid scenarios as well.

Reply:
Added tests for invalid scenarios.

---
In reply to: [447344980](https://github.com/dotnet/roslyn/pull/45396#discussion_r447344980) [](ancestors = 447344980)